ironfish/src/blockchain/blockchain.ts

```diff
@@ -1284,10 +1355,6 @@ export class Blockchain<
       return
     }
 
-    if (!this.head) {
```

Comment:
Would you mind bringing this back so we don't have to know all the conditions to call this function? It also simplifies the call sites.

Reply:
This check won't ever be hit now that `this.head` throws if it's null. We don't have any checks around the callsite for this as far as I know

Reply:
well I was thinking moving the check in open for _head but that's fine